src/base/bittorrent/session.cpp

```diff
@@ -1808,14 +1841,14 @@ void Session::processShareLimits()
             }
 
             if (torrent->seedingTimeLimit() != TorrentHandle::NO_SEEDING_TIME_LIMIT) {
-                const int seedingTimeInMinutes = torrent->seedingTime() / 60;
-                int seedingTimeLimit = torrent->seedingTimeLimit();
+                const auto seedingTimeInMinutes = std::chrono::duration_cast<std::chrono::minutes>(torrent->seedingTime());
```

Comment:
putting "InMinutes" in the variable name here smells a bit like hungarian notation now. It's already encoded in the type

Reply:
These changes were long living as an unofficial patch in my fork. I did want to change as less as possible to simplify merging for me, and did not change the function name. That is why this PR is full of similar inappropriate for the mainline decisions. I hope to sort that out before merging.